utilities/EluvioLiveCli.js

```diff
@@ -762,7 +769,7 @@ yargs(hideBin(process.argv))
           describe: "Target address to mint to",
           type: "string",
         })
-        .option("quanity", {
+        .option("quantity", {
```

Comment:
@elv-wayne not sure how this worked before (?)

Reply:
oh weird!